src/functions/publishAppFunctions.ts

```diff
@@ -91,11 +92,16 @@ const uploadAppFunctions = async (
       );
     }
 
-    const { created, updated, deleted } = (await res.json()) as PublishResponse;
+    const { created, updated, deleted, compiled } =
+      (await res.json()) as PublishResponse;
 
     created.forEach((result) => logResult(result, 'Create'));
     updated.forEach((result) => logResult(result, 'Update'));
     deleted.forEach((result) => logResult(result, 'Delete'));
+    if (config.compile) {
```

Comment:
How is `config.compile` set to true?

Reply:
Via the `-c` cli flag or manually in config.json or in `~/.bb-cli.json`.
This was covered in a previous MR that already went to master.